src/lib/components/LightSettingsPanel.svelte

```
@@ -0,0 +1,775 @@
+			}
+			const result = await parseJobResponse(response);
+			if (result.status !== 'processing') throw new Error('invalid_response');
+			request.setActiveLightSettingsJob(result.id, sourceRender, instruction);
+			if (
+				navigatedAwayWhileSubmitting ||
+				!isEditToolRoute(page.route.id, page.url.searchParams, 'light-settings')
+			) {
+				return;
+			}
+			try {
+				await goto(buildWorkspaceUrl('edit', request, { tool: 'light-settings' }), {
```

ESLint

[error] Unexpected goto() call without resolve().

(svelte/no-navigation-without-resolve)